doc/man3/ASN1_STRING_length.pod

```diff
@@ -28,11 +28,11 @@ ASN1_STRING_to_UTF8 - ASN1_STRING utility functions
 
 These functions allow an B<ASN1_STRING> structure to be manipulated.
 
-ASN1_STRING_length() returns the length of the content of I<x>.
+ASN1_STRING_length() returns the length of the content of I<x>. I<x> B<MUST NOT> be NULL.
```

Comment:
I checked some other places among the pod files, NULL should also be bold as "B\<NULL>"

Reply:
No! It should not.